test/agentchat/contrib/capabilities/test_image_generation_capability.py

```diff
@@ -0,0 +1,203 @@
+import itertools
+import os
+import tempfile
+from typing import Any, Dict, Tuple
+
+import pytest
+from conftest import skip_openai  # noqa: E402
+from PIL import Image
+from test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST  # noqa: E402
+
+from autogen import code_utils
+from autogen.agentchat.contrib.capabilities import generate_images
+from autogen.agentchat.contrib.img_utils import get_pil_image
+from autogen.agentchat.conversable_agent import ConversableAgent
+from autogen.agentchat.user_proxy_agent import UserProxyAgent
+from autogen.cache.cache import Cache
+from autogen.oai import openai_utils
+
+try:
+    from openai import OpenAI
```

Comment:
No need to test for import error of openai, because it is now a required dependency. Just use `skip_openai`.

Reply:
Actually, we can remove openai import as it is not used.